src/woo_publications/urls.py

```diff
@@ -36,6 +37,7 @@
         name="password_reset_done",
     ),
     # Use custom login views for the admin + support hardware tokens
+    path("admin/login/failure/", AdminLoginFailure.as_view(), name="admin-oidc-error"),
     path("admin/", include((urlpatterns, "maykin_2fa"))),
     path("admin/", include((webauthn_urlpatterns, "two_factor"))),
     path("admin/hijack/", include("hijack.urls")),
```

Comment:
please organize all the admin URLs properly in `woo_publications.admin.urls`, like here:

https://github.com/open-formulieren/open-forms/blob/97eab0196e3605ff7b9afb499079ee9c6ca18b4e/src/openforms/urls.py#L43